tools/uswds-color-tokens-figma.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: USWDS Color Token Template 
+nav: Tools
+permalink: /tools/uswds-color-token-template/
+sidenav: tools-side-nav
+---
+# {{ page.title }}
+
+<div class="iframe-container">
```

Comment:
I see a login prompt in the iframe. Is there any way to avoid this?
Also, the bottom border is cut off for some reason.

Reply:
https://www.evernote.com/l/ACRm_yDrKm9IVY-T9K8duNopu55roDVsAh8

Reply:
Okay. I want to embed this view: https://www.figma.com/community/file/1024394345443465950 but I'm having trouble figuring that out. Do you think it would work if I embed a screen shot and then link the screen shot image to this community file?